### ISSUE TITLE
Loss Landscape Dependent Self-Adjusting Learning Rates in Decentralized Stochastic Gradient Descent

### ISSUE THREAD

**Issue by Hiroki11x** [Hiroki11x/Papers#169]
## 一言でいうと
LAMBに勝ってる？？

## 概要
Synchronous Stochastic Gradient Descent (SSGD) 

<img width="463" alt="Screenshot 2021-12-07 at 10 50 56 AM" src="https://user-images.githubusercontent.com/8721858/145062288-747be2eb-1978-4db1-bedc-a7077de7f155.png">
<img width="482" alt="Screenshot 2021-12-07 at 10 51 11 AM" src="https://user-images.githubusercontent.com/8721858/145062289-f66bd2cf-3891-4b4d-8dfe-ebf7a9e76e6b.png">
<img width="607" alt="Screenshot 2021-12-07 at 10 49 49 AM" src="https://user-images.githubusercontent.com/8721858/145062293-082cdbca-5371-4a28-be8d-86efe7a41462.png">

大規模な深層学習（DL）の学習には、分散型深層学習（DDL）が不可欠です。同期式確率的勾配降下法（SSGD）1は、事実上のDDL最適化手法です。DDLの実行速度を向上させるには、十分に大きなバッチサイズを使用することが重要です。バッチサイズが大きい場合，パラメータ更新回数の減少を補うために，学習率を上げる必要があります．しかし、学習率を大きくすると、SSGDの収束性が損なわれ、学習が発散しやすくなる可能性があります。近年、分散学習速度を向上させるために、Decentralized Parallel SGD (DPSGD)が提案されている。本論文では、DPSGDが大規模なバッチ設定において、システム的な実行時間の利点だけでなく、SSGDに比べて大幅な収束性の利点があることを発見した。DPSGDの学習ダイナミクスを詳細に分析した結果、DPSGDはランドスケープに依存した追加のノイズを導入し、収束性を向上させるために有効な学習率を自動的に調整することがわかりました。さらに、このノイズが損失分布を平滑化し、より大きな学習率を可能にすることを理論的に示します。本研究では、18の最先端のDLモデル／タスクを対象とした大規模な研究を行い、大規模なバッチ設定において、SSGDが大きな学習率で発散するケースでも、DPSGDはしばしば収束することを実証しました。この結果は、2つの異なるアプリケーション・ドメインで一貫しています。この結果は、コンピュータビジョン（CIFAR10およびImageNet-1K）と自動音声認識（SWB300およびSWB2000）という2つの異なるアプリケーションドメイン、および2つの異なるタイプのニューラルネットワークモデルで一貫しています。畳み込み型ニューラルネットワークと長短記憶型リカレントニューラルネットワークです。
